puppet/zulip_ops/files/tornado_process_exporter.yaml

```diff
@@ -0,0 +1,6 @@
+process_names:
+  - name: "tornado-{{.Matches.Port}}"
+    comm:
+      - python3
+    cmdline:
+      - runtornado\s+(?P<Port>\S+)
```

Comment:
I don't know the syntax here, but I think it's almost always `127.0.0.1:{port}; not sure whether this is the best pattern.

Reply:
Yeah, calling the variable "port" is probably not the best.  runtornado technically could accept `host:port` or `port` arguments, but that gets hard to name.

I've switched it to only find things on 127.0.0.1, since that's all that the supervisor config will ever start.